src/amber/cmd/templates/controller/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -1,8 +1,12 @@
 class <%= @name.capitalize %>Controller < ApplicationController 
-<% @actions.each do |action, verb| -%>
-
-  def <%= action %>
-    render("<%= action %>.<%= @language %>")
+<%= 
+  actions = @actions.map do |action, verb| 
+    <<-ACTION
```

Comment:
I kinda like how it was but all good :)

Reply:
Lol. Yeah the template definitely looked better before. The generated code looks better now though and I think that's more important.